README.md

```diff
@@ -294,6 +294,26 @@ $ cd android-components
 $ ./gradlew assemble
 ```
 
+### Configuring the Android SDK ###
+
+You will need the Android SDK installed (i.e. pointed to by the `ANDROID_SDK_ROOT` environment variable), and configured to indicate acceptance of the Android SDK licenses. This can be accomplished on Linux as follows:
+
+```
```

Comment:
I don't think we should include command-line examples that point to a particular version numbers because it can lead to people copying the same contents are pasting it when these versions go out of date quickly. It also means that we have to maintain more documentation to keep the version numbers and hashes up-to-date.

How about we point to the official documentation on how to setup the SDK if we need to a section for it?

Reply:
I did link a pretty old SDK version. Since I made this PR I've switched to installing the Command Line Tools and letting Gradle use them to install the SDK itself.

There isn't quite official documentation on how to install the Command Line Tools that I can find. Google mostly just wants you to use Android Studio, and all their articles on installing things already have their command line tools to install stuff with.

The command line tools download is at https://developer.android.com/studio#downloads under "Command line tools only", and the docs on how to use the sdkmanager it provides are [here](https://developer.android.com/studio/command-line/sdkmanager), but they don't seem to walk you through setting up the directory structure that makes it work. 

How about this approach: it fetches the newest Command Line Tools download URL at runtime by parsing it out of Google's site (because there's no stable URL for the most recent version).

```
mkdir android-sdk-linux
cd android-sdk-linux
mkdir -p licenses
echo "8933bad161af4178b1185d1a37fbf41ea5269c55" >> licenses/android-sdk-license
echo "d56f5187479451eabf01fb78af6dfcb131a6481e" >> licenses/android-sdk-license
echo "24333f8a63b6825ea9c5514f83c2829b004d1fee" >> licenses/android-sdk-license
mkdir cmdline-tools
cd cmdline-tools
wget "$(curl -s https://developer.android.com/studio | grep -oP "https://dl.google.com/android/repository/commandlinetools-linux-[0-9]+_latest.zip")"
unzip commandlinetools-linux-*_latest.zip
cd ..
```